### ISSUE TITLE
Docker image is out-of-date

### ISSUE THREAD

**Issue by tmaier** [mailhog/MailHog#204]
https://hub.docker.com/r/mailhog/mailhog/ has been rebuilt one year ago. This is quite some time.

The repository is already an automated build repository.
You can link it to the repository of the base image. It would then get rebuilt when the linked repository got built.

To do so go to
https://hub.docker.com/r/mailhog/mailhog/~/settings/automated-builds/
scroll down to "Repository Links" and add `alpine` to the list

**Comment by Mairu** [comment]
Last commit to master was about one year ago :wink:

**Comment by mld** [comment]
Yeah, but there's a bunch of updates to Alpine since then, which seems to be @tmaier's point.

**Comment by tmaier** [comment]
Correct, @mld.

Sorry for not making this point clearer.

People often miss the added responsibility when we compare Caas with PaaS

**Comment by tyndyll** [comment]
There are a few PR's in with Dockerfile updates. It's likely that they will be accepted (we're just testing them) at which point I'll close this

**Comment by Nklya** [comment]
There is another problem with docker images.
They are not tagged on Docker Hub.

<img width="436" alt="image" src="https://user-images.githubusercontent.com/26086394/50215073-9fa0da80-038a-11e9-8399-e6ef6a37ba9d.png">

P.S. I can create separate issue if needed

**Comment by webdobe** [comment]
This is still an issue and yet alpine is considered to have one of the most vulnerable 0 days in docker? https://blog.aquasec.com/cve-2019-5021-alpine-docker-image-vulnerability 3.4 the image this is built from has no fix available.  This is bad news guys.

**Comment by tyndyll** [comment]
A change was made earlier today to use the official Golang image. A push to Dockerhub is imminent

**Comment by bradbulger** [comment]
I happened to be tailing the logs of my apache container in my local setup and saw a ton of obvious hackish scans that were originating from the MailHog container IP. I am rebuilding now, but could this be related to these security concerns?

**Comment by webdobe** [comment]
Any update on this?  Dockerhub is still not updated.
